CMakeLists.txt

```diff
@@ -148,6 +148,10 @@ if(CMAKE_COMPILER_IS_GNUCXX)
     # Enable a bunch of compiler warnings...
     # http://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wshadow -Wconversion -Wcast-qual -Wformat=2")
+    # GCC 8 and above has a new heuristic to detect invalid pointer
+    # casts. However, this results in false positives on the Python
+    # C API which we compile against to generate Python bindings.
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-cast-function-type")
```

Comment:
The warning disabling only applies to the Python bindings, so it should be in the `src/pyblue/CMakeLists.txt` file. The idea is to control which code really needs it to avoid hiding some potential issue on other parts of the library.

Reply:
Good point. I've moved it to `src/pyglue/CMakeLists.txt`.